bot.py

```diff
@@ -47,8 +57,11 @@ async def on_message(message: discord.Message):
         await message.delete()
         await message.channel.send(link_fixer.handle_message(message))
 
+    blacklist = filter._get_blacklist()
```

Comment:
If we access a method outside of it's original file, it shouldn't be specified as private

Reply:
I'm a bit shaky on my Python conventions, this has been fixed